src/tests/ftest/launch.py

```diff
@@ -27,14 +27,55 @@
 import json
 from multiprocessing import Process
 import os
+import re
+from shutil import rmtree
 import socket
 import subprocess
+from tempfile import mkdtemp
 import time
 import yaml
 
 
 TEST_LOG_FILE_YAML = "./data/daos_avocado_test.yaml"
 BASE_LOG_FILE_YAML = "./data/daos_server_baseline.yaml"
+SERVER_KEYS = (
+    "test_machines",
+    "test_servers",
+    "daos_servers",
+    "servers",
+    "test_machines1",
+    "test_machines2",
+    "test_machines2a",
+    "test_machines3",
+    "test_machines6",
+    )
+CLIENT_KEYS = (
+    "test_clients",
+    "clients",
+    )
+
+
+class TemporaryDirectory(object):
+    # pylint: disable=too-few-public-methods
+    """Create and return a temporary directory.
+
+    This is a quick and dirty implementation of the tempfile.TemporaryDirectory
+    class available in python 3.2+.  It should be replaced by the following
+    when we move from python 2.7:
+
+        from tempfile import TemporaryDirectory
```

Comment:
Couldn't you actually wrap this definition with a:
```
try:
    from tempfile import TemporaryDirectory
except ImportError:
    class ...
```

to automatically achieve the goal of the comment?

Reply:
Good suggestion, thanks!